components/file-explorer/viewers/base.tsx

```
@@ -1,52 +1,127 @@
+    const Content = useMemo(() => {
+        switch (tab.viewerType) {
+            case ViewerType.NOT_IDENTIFIED_YET:
+                return (
+                    <div className="flex justify-center items-center h-full">
+                        <LoaderCircle className="size-4 animate-spin text-muted-foreground" />
+                    </div>
+                )
+            case ViewerType.UNKNOWN:
+                return <LargeViewSelect setType={setType} />
+            default:
+                const Viewer = VIEWERS.find((v) => v.type === tab.viewerType)!.component
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)